workflow-utils/bin/concat-files.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cat $1 $2
```

Comment:
Assuming this is for a cwltool, it seems unnecessary. Can't the tool just specify `cat` as its baseCommand and take two inputs?